opentelemetry-sdk/src/opentelemetry/sdk/metrics/export/batcher.py

```diff
@@ -60,17 +60,15 @@ def process(self, record) -> None:
         """Stores record information to be ready for exporting."""
         # Checkpoints the current aggregator value to be collected for export
         aggregator = record.aggregator
-        aggregator.take_checkpoint()
 
         # The uniqueness of a batch record is defined by a specific metric
         # using an aggregator type with a specific set of labels.
-        key = (record.instrument, aggregator.__class__, record.labels)
+        key = (record.instrument, aggregator.__class__, get_dict_as_key(aggregator.config), record.labels)
```

Comment:
It would be nice if you could update the above comment as well as indicate why we need the extra parameter for uniqueness.

Reply:
done